pkg/scheduler/scheduler.go

```diff
@@ -647,6 +647,9 @@ func (sched *Scheduler) scheduleOne() {
 	}
 	// bind the pod to its host asynchronously (we can do this b/c of the assumption step above).
 	go func() {
+		metrics.SchedulerGoroutines.WithLabelValues("binding").Inc()
```

Comment:
We create a lot of goroutines in the following methods

https://github.com/kubernetes/kubernetes/blob/00458855e487b36ec7356038c6dc7e10a5354018/pkg/scheduler/core/generic_scheduler.go#L681-L689

Reply:
Yes, I am more willing to add more stages than just binding.